Week5/solution-5

```diff
@@ -0,0 +1,30 @@
+#  WRITE CSV FILE USING DIRECTLY
+
+import csv
+with open ('E:\SEM-3\GITHUB ASSIGNMENT\student.csv','w') as f:
+    data=csv.writer(f)
+    COL=["sid","sname","city","contact"]
+    C=[[1,"jay","BARDOLI",9879654625],[2,"BHUPEN","BARDOLI",1234567890],[3,"ABHISHEK","VYARA",9874561230],[4,"HIREN","SURAT",9879461230],[5,"ROMAN","AMRELI",3216549870]]
+    data.writerow(COL)
+    data.writerows(C)
```

Comment:
whats difference between writerows() and writerow()?

Reply:
writerow() is used to write a single row (list) of data to a CSV file, while writerows() is used to write multiple rows (lists) of data to a CSV file. writerow() writes one row, while writerows() writes many rows at once.